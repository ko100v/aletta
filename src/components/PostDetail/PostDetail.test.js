import React from 'react'

import { configure, shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'
configure({ adapter: new Adapter() })

import PostDetail from './PostDetail'

test('Render PostDetail', () => {
    const component = shallow(<PostDetail />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
})