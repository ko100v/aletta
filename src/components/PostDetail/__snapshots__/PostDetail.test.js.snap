// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PostDetail 1`] = `
<div>
  <Post />
</div>
`;
